css/interactive-guides/tabbed-editor.css

```diff
@@ -8,14 +8,28 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
-.teContainer>.nav-tabs>li>a.active {
+ .teContainer>.nav-tabs>li>a {
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+    font-size: 1vw;
```

Comment:
We will need to increase the font-size on mobile screen sizes because 1% viewport width is really tiny.

Reply:
I agree....we will try 2.5vw